src/dev/src/app/i18n-a11y/i18n-a11y.demo.ts

```diff
@@ -5,9 +5,11 @@
  */
 import { Component, OnDestroy } from '@angular/core';
 import { ClrCommonStringsService } from '@clr/angular';
+import { commonStringsDefault } from '@clr/core';
+import { ClrCommonStrings } from '@clr/angular';
```

Comment:
Is `ClrCommonStrings` still (or duplicated) in `@clr/angular`?

Reply:
Still in `@clr/angular` but not duplicated. Extends the plain class so it can be used with Angular's DI system. https://github.com/vmware/clarity/pull/3920/files#diff-7d43362ba399d8b126da4c4c8089e822R12